src/Symfony/Component/Form/Extension/Core/Type/CountryType.php

```diff
@@ -13,11 +13,12 @@
 
 use Symfony\Component\Form\AbstractType;
 use Symfony\Component\Form\ChoiceList\ArrayChoiceList;
-use Symfony\Component\Form\ChoiceList\Loader\ChoiceLoaderInterface;
+use Symfony\Component\Form\ChoiceList\Loader\IntlCallbackChoiceLoader;
 use Symfony\Component\Intl\Intl;
+use Symfony\Component\OptionsResolver\Options;
 use Symfony\Component\OptionsResolver\OptionsResolver;
 
-class CountryType extends AbstractType implements ChoiceLoaderInterface
+class CountryType extends AbstractType
```

Comment:
I would not drop the `implements` here. Technically, that's a BC break.

Reply:
Reverted.